tensorrt_llm/_torch/pyexecutor/py_executor_creator.py

```
@@ -375,21 +383,69 @@ def create_py_executor(
+                "KV connector is only supported with guaranteed no evict scheduler policy."
+            )
+
+        try:
+            module = importlib.import_module(
+                kv_connector_config.connector_module)
+            worker_cls = getattr(module,
+                                 kv_connector_config.connector_worker_class)
+            scheduler_cls = getattr(
+                module, kv_connector_config.connector_scheduler_class)
+
+            # Some connector API implementations may need to establish out-of-band communication between the scheduler and workers.
```

Ruff (0.12.2)

Line too long (131 > 120)

(E501)